teamworkApp/config/routes.rb

```diff
@@ -1,9 +1,16 @@
 Rails.application.routes.draw do
+
+
   root 'high_voltage/pages#show', id: 'home'
+
+  post '/python/importData/' => 'python#importData'
+
   get '/python/studentGraph/' => "python#studentGraph"  
   get '/python/overallBar/' => 'python#overallBar'
-  get '/python/importData/' => 'python#importData'
+  # get '/python/importData/' => 'python#importData'
```

Comment:
Why is this commented out?

Reply:
That was me, it gets used as a put rather than a get, I will delete the line.